src/components/Details.js

```diff
@@ -1,7 +1,72 @@
 import React from 'react';
+import { useParams, Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { BiLeftArrow } from 'react-icons/bi';
```

Comment:
Good choice using react-icon!

Reply:
Thank you for reviewing our code brother.